client/src/mocks/handlers/unleashHandler.ts

```diff
@@ -0,0 +1,7 @@
+import { http, HttpResponse } from "msw";
+
+export const unleashHandler = [
```

Comment:
Mocker denne unleash? Hvordan velger man å bruke den? :)

Reply:
Jepp, for å kunne teste enabla flag lokalt :)
For å legge til nytt flagg, skriv feks: { lokalFlag: false, nyttFlagg: true }

så vil

const {flag} = useFlag("nyttFlagg")

gi {flag.enabled} som true i komponent

Reply:
og 
const {flag} = useFlag("lokalFlag")
vil gi {flag.enabled} som false?